Makefile.mk

```diff
@@ -0,0 +1,19 @@
+PHONY: build-run clean logs stop help
```

Comment:
@pankajastro it looks a bit weird that we have both a `Makefile` and a `Makefile.mk` in the project's root.
What do you think of the following directory structure:
```
astro-sdks/
    Makefile
    mk/
        local.mk
        container.mk
```
The Makefile could include includes to both local and container subsets, something similar to what is described in:
https://downloads.haskell.org/~ghc/6.4/docs/html/building/sec-makefile-arch.html

Reply:
I have tried here df5c6616100121cf7f33a5e1fa4f423f352a97c1 please let me know how it is looking.